esmvalcore/preprocessor/_time.py

```diff
@@ -284,12 +314,15 @@ def hourly_statistics(cube, hours, operator='mean'):
         iris.coord_categorisation.add_year(cube, 'time')
 
     operator = get_iris_analysis_operation(operator)
-    cube = cube.aggregated_by(['hour_group', 'day_of_year', 'year'], operator)
+    result = cube.aggregated_by(['hour_group', 'day_of_year', 'year'],
+                                operator)
 
-    cube.remove_coord('hour_group')
-    cube.remove_coord('day_of_year')
-    cube.remove_coord('year')
-    return cube
+    result.remove_coord('hour_group')
+    result.remove_coord('day_of_year')
+    result.remove_coord('year')
+
+    _aggregate_time_fx(result, cube)
+    return result
```

Comment:
what's this little function doing - is it a placeholder you forgot to remove?

Reply:
nevermind, am silly - I thought this was outside the previous function, and this sorts out my point of returning cube too, which is rightly returned as result